example/pubspec.yaml

```diff
@@ -10,19 +10,19 @@ description: Demonstrates how to use the square_reader_sdk plugin.
 version: 1.1.2+1
 
 environment:
-  sdk: '>=2.12.0 <3.0.0'
+  sdk: '>=2.14.0 <3.0.0'
 
 dependencies:
   flutter:
     sdk: flutter
 
-  permission_handler: ^8.0.0+2
+  permission_handler: any
```

Comment:
Is that a good idea to pick any version of `permission_handler`?

Reply:
yes we don't need to give specific version. it will take latest version automatically.

Reply:
can the latest version break the app unexpectedly?

Reply:
I changed version any to specific version.